bare_metal_how_to.md

```diff
@@ -0,0 +1,176 @@
+# MVP - Bare metal provisioning
+
+## Preparation
+
+### Setup - Ironic server
+
+To be used as Ironic server the host's requirements are:
+
+1. CentOS Stream 9
+2. root access
+
+#### Install
+
+<!-- TODO: update after https://gitlab.cee.redhat.com/pnt-resource-hub/rhub-baremetal/-/merge_requests/11 -->
+
+Run playbook available on: https://gitlab.cee.redhat.com/pnt-resource-hub/rhub-baremetal
+
+```shell
+$ ansible-playbook -i inventory/dsal.yml playbooks/bifrost_setup.yml --extra-vars "target=setup<N>"
+```
+
+Get the password:
```

Comment:
Might want to explain why they need to get this password.  Where is it needed later?